cdap-api/src/main/java/co/cask/cdap/api/customaction/AbstractCustomAction.java

```diff
@@ -58,6 +61,7 @@ protected void setProperties(Map<String, String> properties) {
   }
 
   @Override
+  @TransactionPolicy(TransactionControl.IMPLICIT)
```

Comment:
Yes, I added that to make it clear what the default is for each program type. The program runners do use the same as default.